test/aspnettests.js

```diff
@@ -115,10 +115,10 @@ describe('ASP.NET project file creation when kestrel command exists', function (
     before(function (done) {
         helpers.run(path.join( __dirname, '../generators/app'))
         .inTmpDir(function(dir) {
-            createTestProjectJson(dir, 'EXISTING_KESTREL_COMMAND'); })
+            createTestProjectJson(dir, 'EXISTING_WEB_COMMAND'); })
         .withLocalConfig(function() {
             return { "appInsightsOptIn": false, "runningTests": true }; })
-        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-beta8' })
+        .withPrompts({ projectType: 'aspnet', aspNetVersion: '1.0.0-rc1-final' })
```

Comment:
But this is an internally stored value that get re-mapped to rc1-update1 in getDockerImageName, why not just use the version we want to start with?